### ISSUE TITLE
Failed to generate config file

### ISSUE THREAD

**Issue by shinokada** [inlang/inlang#660]
## Problem

When I run `npx @inlang/cli config init` on a fresh SvelteKit, I get an error:

```
➜  i18n-sveltekit npx @inlang/cli config init
Need to install the following packages:
  @inlang/cli@0.6.0
Ok to proceed? (y) y
✖ Failed to generate config file.
undefined
```

### Expected behavior

It should install inlang.

## Steps to reproduce

```
npm create svelte@latest i18n-sveltekit
cd i18n-sveltekit
pnpm I
npx @inlang/cli config init
```

### Additional information (screenshots?)
<img width="511" alt="image" src="https://user-images.githubusercontent.com/147320/236716954-f6a25ec6-09d6-47b5-a51c-fdb2f26b3933.png">

**Comment by samuelstroschein** [comment]
Thanks for filing! Will be solved as part of https://github.com/inlang/inlang/issues/544

**Comment by shinokada** [comment]
<img width="670" alt="image" src="https://github.com/inlang/inlang/assets/147320/14bae11b-abb3-4035-8ffa-0ba41bfa966a">

This is not solved and should be reopen.

**Comment by samuelstroschein** [comment]
Ran into this myself. The server is not up to date. Hopefully fixed with https://github.com/inlang/inlang/commit/78d93e696dee1e507996bd667b584c4f4c3777fc

**Comment by samuelstroschein** [comment]
Will follow up if this is fixed

**Comment by samuelstroschein** [comment]
Should be fixed by #811. FYI using chatgpt for the config generation was probably not worth it though and will be replaced in the future.
